djoser/webauthn/views.py

```diff
@@ -29,8 +30,9 @@
 User = get_user_model()
 
 
-class SingupRequestView(APIView):
+class SignupRequestView(APIView):
     permission_classes = (AllowAny,)
+    serializer_class = Serializer
```

Comment:
Shouldn't it set the serializer class used in the `post()` method, in this case `WebauthnSignupSerializer`?

Reply:
Yes, I think this would be correct. Fixed